config/gh-actions/deploy-surge.sh

```diff
@@ -51,15 +60,15 @@ do
   echo "Deploy domain: ${MARKBIND_DEPLOY_DOMAIN}"
   surge --project ${MARKBIND_DEPLOY_PATH} --domain $MARKBIND_DEPLOY_DOMAIN;
 
-  if [ "$TRAVIS_PULL_REQUEST" != "false" ] # only create github statuses when it is a PR
+  if [ "$ACTION_IS_PULL_REQUEST" != "false" ] # only create github statuses when it is a PR
   then
     # Create github statuses that redirects users to the deployed dashboard and markbind docs
-    curl "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/statuses/${TRAVIS_PULL_REQUEST_SHA}?access_token=${GITHUB_API_TOKEN}" \
+    curl "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/statuses/${ACTIONS_PULL_REQUEST_HEAD}?access_token=${GITHUB_TOKEN}" \
     -H "Content-Type: application/json" \
     -X POST \
     -d "{\"state\": \"success\",\"context\": \"dashboard/surge/deploy/${DEPLOY_SUBDOMAIN}\", \"description\": \"Deploy domain: ${DASHBOARD_DEPLOY_DOMAIN}\", \"target_url\": \"${DASHBOARD_DEPLOY_DOMAIN}\"}"
 
-    curl "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/statuses/${TRAVIS_PULL_REQUEST_SHA}?access_token=${GITHUB_API_TOKEN}" \
+    curl "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/statuses/${ACTIONS_PULL_REQUEST_HEAD}?access_token=${GITHUB_TOKEN}" \
```

Comment:
Meanwhile, can you check if the repo owner here is changed due to our previous checkout command? (our new feature)

The `not found` command should come from here.

Reply:
Adding another environment variable should solve the problem.